### ISSUE TITLE
Allow Commenting and emoji expession on closed issues on all Godot repositories

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot-proposals#2561]
### Describe the project you are working on
This applies to anything Godot related

### Describe the problem or limitation you are having in your project
I would like to express my opinion on an issue or comment, but I can't because the issue has been closed.
I have been in this situation many times before, but last time here:
https://github.com/godotengine/godot-docs/pull/4799
I have missed the opportunity to express myself by merely a hand full of days. This is sadly quite ironic as this issue is about how the Godot project communicates community participation and governance.
Opening up a new issue is not the solution, as this just creates gigantic overhead that is harder and harder too keep track of as such a discussion spreads over multiple issues. 
Opening up a new issue is also not a solution if all I want is to thumb up a comment.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
It would give me as someone coming late to these discussions a stronger feeling of being a community member and that my voice is welcomed and heard.
It lessen the emotional impact of closing an issue for the person opening it as well as for others who feel strongly about it (Usually people open issues because they care about something.)
It will reduce the stress of having to keep up with every issue. If the issue has been closed and many people still thumb it up, there can be a decission to reopen it or open another issue. By disallowing any expression of support, this is impossible. The only chance for expression would be make another issue (stating the same things?). **But who does that if they just saw the core team closing such an issue?**

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
Just make sure comments and emoji on issues and comments of issues are allowed on all Godot repos even if they are closed.

### If this enhancement will not be used often, can it be worked around with a few lines of script?
It's a Github/Community meta proposal.

### Is there a reason why this should be core and not an add-on in the asset library?
It's a Github/Community meta proposal.

**Comment by LightningAA** [comment]
The issue you referenced was locked on purpose, most closed issues aren't locked

**Comment by golddotasksquestions** [comment]
Not sure what the purpose is of locking a discussion by the community about how community participation is expressed externally.

The optics of this are rather counter productive.

https://github.com/godotengine/godot-proposals/issues/100 btw has been locked 1,5 years now. Without the chance for anyone to express agreement or disagreement. How long of a cooldown does a discussion about an input buffer need?
Why does a discussion how the community participation is worded need to be locked down?

**Comment by pycbouh** [comment]
I'm sorry, but you're asking us not to use a moderation tool designed specifically to prevent heated discussions, and this is not going to happen. Tool exists for a purpose and we only use it for that purpose. And sometimes to prevent users from abusing the open nature of the project by spaming requests in various forms.

> Why does a discussion how the community participation is worded need to be locked down anyway?

You can ask that question at [Godot Contributors Chat](https://chat.godotengine.org/). You can even ask to open up issue X, as some people sometimes do and [we sometimes agree to](https://github.com/godotengine/godot-proposals/issues/1866). This can be resolved amiably on the case by case, but as it stands, this proposal is not going to be accepted. We will use the moderation tool for moderating discussions in future.

**Comment by golddotasksquestions** [comment]
You are making this worse.

**Comment by pycbouh** [comment]
The discussion is open, you are free to continue it. I've closed it merely because there is no reason to expect us not to use a moderation tool on a public forum. Whether you have a problem with a specific issue being closed or not, this proposal is broad and cannot be accepted in this form. And for specific issues you don't need to make proposals, you can just talk to us on the appropriate platform.

Thank you.

**Comment by golddotasksquestions** [comment]
Noone asked for substaining moderation. I most certainly did not. I did **not** ask for locks on threads not to be used.
I am asking for allowing to allow commenting and expression with emojis in principal on all repos. That's all there is to it.

Like I can do it here now as well.

If this is already possible, than yes, let's close this issue here!
But if it is not possible, you closing it and referring me else where is EXACTLY why people who are still here ask for a change how community is handled.

If you don't see this, who will?

**Comment by pycbouh** [comment]
> I did not ask for locks on threads not to be used. I am asking for allowing to allow commenting and expression with emojis in principal on all repos.

This is what locking is: disabling commenting for non-collaborators. And, by GitHub's own wisdom, disabling emote reactions as well. We don't explicitly disable reactions, it comes with locking.

So I'm not entirely sure what you propose then, because what you ask is related to locking, yet you say you don't mind locking.

**Comment by akien-mga** [comment]
> Noone asked for substaining moderation. I most certainly did not. I did not ask for locks on threads not to be used.
I am asking for allowing to allow commenting and expression with emojis in principal on all repos. That's all there is to it.

Those two statements are equivalent though. Locked threads **is** the moderation feature that makes you enable to comment and react on some specific threads (that is to say, the threads which have been locked). 99.9% of issues don't get locked when closed. Locking an issue is a very rare action which is taken by moderators, usually for valid reasons. You may disagree with them, but that's best expressed on the Godot contributors chat and not on a formal proposal.

---

So the only way this proposal can be read and actioned upon is "why was https://github.com/godotengine/godot-docs/pull/4799 locked, and can it be reopened?".

That's @reduz's decision, and it was probably badly conveyed (I wouldn't have locked the PR myself), but the point is that we're working on a proper governance document that fully supersedes that PR. Defining and documenting the project's governance is the role of the people who are in charge of the project, and this is not something which should be designed by committee in a PR. That locked PR also started adding some not-so-friendly ping-pong bikeshedding around specific semantic questions which were not relevant to defining the project's governance.

In short - please be patient, we're working on an exhaustive document that describes the governance. Once it's ready, it will be up for review and you'll have the opportunity to give feedback.

**Comment by golddotasksquestions** [comment]
@akien-mga 
You (and maybe pycbouh as well) completely misunderstood me as well as the intention and purpose of this proposal.

I noticed that the [linked issue](https://github.com/godotengine/godot-docs/pull/4799) was closed and I could not participate. 

Is this only the case if an issue was manually locked?

If so this proposal is void.

pycbouh however did not say tell me so far and you did neither. When pycbouh closed my proposal here he did without saying anything about whether or not issues are locked by default when closed.

Again, if issues are not locked by default when closed, then this proposal is void! But closing this mere minutes after I opened this proposal without any reference to the issue I am describing, while I follow the proposal rules and referring me elsewhere while this is about Github and I'm writing a Github issue  - really sucks. 

This sucks even more considering the late discussion on how community efforts are handled.
If you guys don't see the problem with this, who else will?

**Comment by LightningAA** [comment]
Issues are not locked by default.

**Comment by golddotasksquestions** [comment]
"most closed issues aren't locked" to me sounds like in _some_ repos closed issues **are** locked by default.
If they are not, this proposal can be safely closed and ignored

**Comment by pycbouh** [comment]
No, there is no rule for any repo to lock the issues. No issues are locked automatically or as a part of the process by members who close them. Issues are only locked when a member or a group of members decide that it's unsafe or unproductive to keep it open for public discussion. A closed issue is not a locked issue.

So if that was your concern, fear not.

**Comment by Xrayez** [comment]
Oh, I've stumbled upon this issue and I got slightly surprised noticing the linked PR! I'd likely comment on this earlier but this issue got closed almost immediately after creation.

So, since reduz deleted my comment there and the PR got locked, I'd just like to take this opportunity here and say my reaction to this (publicly, on the most open platform out of them all, unlike chat rooms, because we're talking about community here...)

I'm not sure whether me or aaronfranke would have any issue with this if only the governance model was clear from the moment Godot got open-sourced. I mean, that's the first thing you'd have to do as a project founder to avoid having discussions like this pop up in the first place.

Akien summed it up:

> That's reduz's decision, and it was probably badly conveyed

Even when I disagree with Akien or he disagrees with me, it seems like we always tend to try to understand each other anyway, and I highly appreciate this attitude. So, my only wish is having better communication and desire to understand other people. Less premature decisions, more understanding, which lead to better relationships and trust.

Unfortunately, I'm unlikely to gain trust within the community of the core developers due to misunderstandings like this. If expressing explicit disapproval leads to situations like this, then I'm not sure what to say, it feels like just being silent is the best solution to avoiding conflicts. But how is it possible to progress otherwise, both as a project and as an individual?

My respect to everyone who dedicate their lifetime moderating these issues and improving the engine on a daily basis. Everybody have their home and people in general are very smart coming up with whatever game rules, but it's necessary that other people know about those rules in order to have a fair game. That said, we have to know what to say and what not to say, it should *not* be a guessing game, don't you think? I'm aware that we have a code of conduct, but I don't think I've breached anything throughout my time contributing to the engine, nor other people who participated in those recent discussions, there should be better justifications in order to be able to close and lock issues like this.

That's probably my last post regarding this topic, sorry if you're disturbed having this in your mail box, because sometimes I think that's one of the reasons why issues get closed and locked.

Nonetheless, thanks for reading! I hope you better understand my opinion and my worldview, and I'd like that the rest of the community know about my opinion as well, not just a handful of the core developers.
